src/Servers/IIS/IIS/test/Common.FunctionalTests/Inprocess/StartupTests.cs

```diff
@@ -183,18 +183,15 @@ public async Task HelloWorld(TestVariant variant)
             await StartAsync(deploymentParameters);
         }
 
-        [ConditionalFact(Skip="https://github.com/aspnet/AspNetCore/issues/3950")]
+        [ConditionalFact]
         [RequiresIIS(IISCapability.PoolEnvironmentVariables)]
         public async Task StartsWithPortableAndBootstraperExe()
         {
             var deploymentParameters = _fixture.GetBaseDeploymentParameters(_fixture.InProcessTestSite, publish: true);
             // We need the right dotnet on the path in IIS
             deploymentParameters.EnvironmentVariables["PATH"] = Path.GetDirectoryName(DotNetCommands.GetDotNetExecutable(deploymentParameters.RuntimeArchitecture));
-
-            // rest publisher as it doesn't support additional parameters
-            deploymentParameters.ApplicationPublisher = null;
             // ReferenceTestTasks is workaround for https://github.com/dotnet/sdk/issues/2482
-            deploymentParameters.AdditionalPublishParameters = "-p:RuntimeIdentifier=win7-x64 -p:UseAppHost=true -p:SelfContained=false -p:ReferenceTestTasks=false";
+            deploymentParameters.AdditionalPublishParameters = "AppHost";
```

Comment:
Seems like we're misusing what `AdditionalPublishParameters` is meant for (msbuild args). I think this deserves a new property: `DeploymentParameters.PublishProfile`

Reply:
It's a bit of layering thing. I don't want to add DeploymentParameters property that would be only used by this specific implementation in IIS.